package.json

```diff
@@ -56,8 +56,9 @@
     "jstz": "~1.0.7",
     "lodash": "~4.15.0",
     "multiselect": "~0.9.12",
+    "prop-types": "^15.5.9",
     "react": "^15.1.0",
```

Comment:
I thought the deprecation warnings just came from 15.5.4 and if we keep the older version features like props-type might conflict? or in other words, i'm uncertain if keeping 15.0 will work with the props-type package?